controller/handler/trial.js

```diff
@@ -88,7 +107,8 @@ function trialView (request, reply) {
                         'Semicompliant',
                         'Noncompliant'
                     ]
-                })
+                }),
+                endDate: processRule(rules)
```

Comment:
use `Array.reduce` to combine the rules together into a single value

ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce